obspy/io/sac/core.py

```diff
@@ -381,7 +381,8 @@ def _internal_read_sac(buf, headonly=False, debug_headers=False, fsize=True,
     :return: A ObsPy Stream object.
     """
     # read SAC file
-    sac = SACTrace.read(buf, headonly=headonly, ascii=False, checksize=fsize)
+    sac = SACTrace.read(buf, headonly=headonly, ascii=False, checksize=fsize,
+                        encoding=kwargs.get('encoding', 'ASCII'))
```

Comment:
Also note that the `_read_sac()` function is the one that is called by the `obspy.core` function and thus the highest level function that should have the `encoding` parameter. (all plug-in entry points are defined in the root level `setup.py`).